core/lib/workarea/configuration.rb

```diff
@@ -821,9 +821,9 @@ def self.setup_defaults
       # recommended as the default policy.
       config.fulfillment_policies = SwappableList.new(
         %w(
-          Workarea::Fulfillment::Policies::Ship
-          Workarea::Fulfillment::Policies::Ignore
+          Workarea::Fulfillment::Policies::Shipping
           Workarea::Fulfillment::Policies::Download
+          Workarea::Fulfillment::Policies::Digital
```

Comment:
> This is a bad name in my opinion. I understand it makes it more sense to people upgrading, but for new implementations it makes no sense because it doesn't do anything. I'm strongly in favor of clarity of the codebase moving forward and supporting upgrades with documentation.

I agree that from the perspective of all apps and plugins moving forward, `Ignore` is a much better name than `Shipping` for a policy that does nothing.

Reply:
One thought I had would be moving that to a separate `workarea-legacy_digital` plugin.

Reply:
`Ignore` is pretty bad IMO because in the business domain, you're never going to ignore fulfilling an item someone purchased.

Reply:
OK, I can see how `Shipping` makes sense here now

Reply:
I agree its completely unnecessary in new builds, there wouldn't be a reason to use ignore. If `Shipping` or `Download` don't _fulfill_ the needs of the build they should add a new policy. I'm not sure I like the idea of creating plugins to support a half-baked legacy feature. Maybe `Defer` would be a better name? Sounds like we could discuss at the UX meeting

Reply:
need to remove this config?